helpme/templates/helpme/ticket_list.html

```diff
@@ -226,7 +226,7 @@ <h3 class="modal-title mt-3">
 		<form id="comment_form" action="{% url 'helpme-api-create-comment' ticket.uuid %}" method="POST" pk="{{ticket.pk}}">
 		  {% csrf_token %}
 		  {{ comment_form|crispy }}
-		  <input type="submit" value="Reply" class="btn px-4 py-2 mr-3 helpme-button">
+		  <input type="submit" value="Reply" class="btn btn-primary font-weight-bold px-4 py-2 mr-3" style="border-radius:25px;">
```

Comment:
maybe a "rounded-pill" class might do what you are looking for.

Reply:
@renderbox yes, thank you! i somehow missed that one when i was looking before. i will make the switch